languages/English/English_global.php

```diff
@@ -2,10 +2,291 @@
+define("LAN_VSTORE_CUSM_042", "Open orders");
+define("LAN_VSTORE_CUSM_043", "Date/Time");
+define("LAN_VSTORE_CUSM_044", "Invoice-Nr. assigned:");
+define("LAN_VSTORE_CUSM_045", "Order Ref-Nr: assigned");
+define("LAN_VSTORE_CUSM_046", "Unable to load order!");
+define("LAN_VSTORE_CUSM_047", "Order not found!");
+define("LAN_VSTORE_CUSM_048", "Unable to inset/update the order!");
+define("LAN_VSTORE_CUSM_049", "Unable to change order status. No Order loaded!");
+define("LAN_VSTORE_CUSM_050", "Unable to update order status.");
+define("LAN_VSTORE_CUSM_051", "Changed [x] from [y] to [z].");
+define("LAN_VSTORE_CUSM_052", "No order loaded. Unable to update order ref!");
+define("LAN_VSTORE_CUSM_053", "Order is already refunded!");
+define("LAN_VSTORE_CUSM_054", "Only orders with status 'Processing', 'On Hold' and 'Complete' can be refunded!");
+define("LAN_VSTORE_CUSM_055", "Missing payment gateway!");
+define("LAN_VSTORE_CUSM_056", "Refunding details not set!");
+define("LAN_VSTORE_CUSM_057", "Refunded");
+define("LAN_VSTORE_CUSM_058", "Order refunded on [x] by [y] ([z])");
+define("LAN_VSTORE_CUSM_059", "The order has been marked as refunded, but the payment method");
+define("LAN_VSTORE_CUSM_060", "doesn't support automatic refunding!\nYou have to do it manually!");
+define("LAN_VSTORE_CUSM_061", "No.");
+define("LAN_VSTORE_CUSM_062", "Price");
+define("LAN_VSTORE_CUSM_063", "No payment method selected!");
+define("LAN_VSTORE_CUSM_064", "No items to checkout!");
+define("LAN_VSTORE_CUSM_065", "No customer data set!");
+define("LAN_VSTORE_CUSM_066", "No shipping data set! <br /> Please fill in the Shipping address");
+define("LAN_VSTORE_CUSM_067", "Order cancelled by user");
+define("LAN_VSTORE_CUSM_068", "Changes successfully saved!");
+define("LAN_VSTORE_CUSM_069", "File successfully downloaded!");
+define("LAN_VSTORE_CUSM_070", "Billing address is missing!");
+define("LAN_VSTORE_CUSM_071", "No items to checkout!");
+define("LAN_VSTORE_CUSM_072", "Something went wrong! Unable to save changes!");
+define("LAN_VSTORE_CUSM_073", "No billing address set");
+define("LAN_VSTORE_CUSM_074", "No Payment Options Set");
+define("LAN_VSTORE_CUSM_075", "Invalid Payment Type");
+define("LAN_VSTORE_CUSM_076", "Unable to save transaction");
+define("LAN_VSTORE_CUSM_077", "Item not found!");
+define("LAN_VSTORE_CUSM_078", "Invoice pdf not found!");
+define("LAN_VSTORE_CUSM_079", "No categories available!");
+define("LAN_VSTORE_CUSM_080", "Refunding failed!");
+define("LAN_VSTORE_CUSM_081", "Order not loaded!");
+define("LAN_VSTORE_CUSM_082", "Invalid number of item_vars!");
+define("LAN_VSTORE_CUSM_083", "Invoice pdf not found!");
+
+define("LAN_VSTORE_CUST_013", "Order updated to ");
+define("LAN_VSTORE_CUST_014", "Order couldn't be updated to ");
+define("LAN_VSTORE_020", "Your order ");
+define("LAN_VSTORE_021", "is complete and you will receive a order confirmation ");
+define("LAN_VSTORE_022", "with all details within the next few minutes by email!");
```

Comment:
We shouldn't have half-sentences inside LANs. Use [x] if you need to include a variable. 
eg. Your order [x] is complete and you will receive an order confirmation with all details within the next few minutes by email.